.ci/pipelines/build.Jenkinsfile

```diff
@@ -37,37 +37,31 @@ pipeline {
                 stage('build') {
                     failFast true
                     parallel {
-                        stage("build and push operator image") {
+                        stage("build and push operator image and manifests") {
                             agent {
                                 label 'linux'
                             }
                             steps {
                                 sh '.ci/setenvconfig build'
-                                sh 'make -C .ci license.key TARGET=build-operator-multiarch-image ci'
+                                sh 'make -C .ci license.key TARGET=ci-release ci'
```

Comment:
`ci-release` runs this:

```
ci-release: clean ci-check build-operator-multiarch-image
```

`ci-check` runs this, which includes the `generate` step:

```
ci-check: check-license-header lint shellcheck generate check-local-changes check-predicates
```

Since we run `ci-check` here https://github.com/elastic/cloud-on-k8s/pull/6149/files#diff-68c3b03af5147c61c49f4b3316dcdd00fc6c3566986ddf0a12f30f3346cb95a5L29, prior to calling these build steps, I was trying to avoid re-running that step.

Reply:
Makes sense until we added the `agent`. So we can call `generate-crds-v1 build-operator-multiarch-image`.